pkg/server/proto/server.proto

```diff
@@ -619,6 +623,12 @@ message Variable {
     HclPos start = 2;
     HclPos end = 3;
   }
+
+  message Ref {
```

Comment:
Nitpick on name: `Ref` to me usually means a database-style foreign key. That's how I've thought of it when we used refs in Waypoint. I feel like this should be something else, like `Value`.

Reply:
Oh, I think I see, you named this this because of `DataSource.Ref`. I named that `Ref` because of Git refs. Thats what git calls the format used to point to a commit. I think variable should still be `Value` or something like that.